astroquery/alma/core.py

```diff
@@ -400,6 +469,28 @@ def _HEADER_data_size(self, files):
 
         return data_sizes, totalsize.to(u.GB)
 
+    def _HEADER_data_size(self, files):
+        """
+        Given a list of file URLs, return the data size.  This is useful for
+        assessing how much data you might be downloading!
+        (This is discouraged by the ALMA archive, as it puts unnecessary load
```

Comment:
is there then an API they prefer?

Reply:
:shrug: this is old code... I think... why is it green here....

Reply:
I'm really confused how `_HEADER_data_size` got duplicated in one of the commits...